tower-util/Cargo.toml

```diff
@@ -23,15 +23,15 @@ categories = ["asynchronous", "network-programming"]
 edition = "2018"
 
 [dependencies]
-tower-service = { version = "=0.3.0-alpha.2", path = "../tower-service" }
-tower-layer = { version = "=0.3.0-alpha.2", path = "../tower-layer" }
+tower-service = { version = "0.3.0", path = "../tower-service" }
+tower-layer = { version = "0.3.0", path = "../tower-layer" }
 pin-project = "0.4"
 futures-util-preview = "=0.3.0-alpha.19"
 futures-core-preview = "=0.3.0-alpha.19"
 
 [dev-dependencies]
 futures-util-preview = "=0.3.0-alpha.19"
-tokio-test = "=0.2.0-alpha.6"
-tokio = "=0.2.0-alpha.6"
-tower = { version = "=0.3.0-alpha.2", path = "../tower" }
-tower-test = { version = "=0.3.0-alpha.2", path = "../tower-test" }
+tokio-test = "0.2.0"
+tokio = "0.2.0"
```

Comment:
do we still need this if it has no features?

Reply:
Yes, I think so, but lemme check.